build/scripts/PullRequestValidation/ValidateInternalWorkitemForPullRequest.ps1

```diff
@@ -0,0 +1,52 @@
+using module ..\GitHub\GitHubPullRequest.class.psm1
+using module ..\GitHub\GitHubIssue.class.psm1
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $PullRequestNumber,
+    [Parameter(Mandatory = $true)]
+    [string] $Repository
+)
+
+# Set error action
+$ErrorActionPreference = "Stop"
+
+<#
+    .Synopsis
+    Validates that the pull request description contains a line that links the pull request to an ADO workitem.
+    .Parameter ADOWorkItems
+    The IDs of the ADO workitems linked to the pull request.
+    .Parameter PullRequest
+    The pull request to validate.
+#>
+function Test-ADOWorkitemIsLinked() {
+    param(
+        [Parameter(Mandatory = $false)]
+        [string[]] $ADOWorkItems,
+        [Parameter(Mandatory = $false)]
+        [object] $PullRequest
+    )
+
+    $Comment = "Could not find a linked ADO workitem. Please link one by using the pattern 'Fixes AB#' followed by the workitem number being fixed."

```

Comment:
This is fine for MS devs, but for non-MS devs might be better to show something like "PRs cannot be merged if not linked to an internal work item. Repo maintainers are responsible for linking PRs. Please do not add any links to internal work items on your own". Too nice? 😄

Reply:
Now I see the job is called "For Microsoft: Validate link to internal work items", so if you think that's enough, I'm also fine with it.